include/internal/libspdm_responder_lib.h

```diff
@@ -743,6 +743,31 @@ libspdm_return_t libspdm_get_response_chunk_send(libspdm_context_t *spdm_context
 
 #endif /* LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP */
 
+#if LIBSPDM_ENABLE_VENDOR_DEFINED_MESSAGES
+/**
+ * Process the SPDM VENDOR_DEFINED_REQUEST request and return the response.
+ *
+ * @param  spdm_context                  A pointer to the SPDM context.
+ * @param  request_size                  size in bytes of the request data.
+ * @param  request                      A pointer to the request data.
+ * @param  response_size                 size in bytes of the response data.
+ *                                     On input, it means the size in bytes of response data buffer.
+ *                                     On output, it means the size in bytes of copied response data
+ * @param  response                     A pointer to the response data.
+ *
+ * @retval RETURN_SUCCESS               The request is processed and the response is returned.
```

Comment:
Delete all these `retval`s.

Reply:
Hi Steven. I did the changes to the macro names but am uncertain about the deletion of the retvals. All the methods above this code, like libspdm_get_response_chunk_get() or libspdm_get_response_csr() have the same retvals. Granted, libspdm_get_vendor_defined_response() can return more status values, like:

LIBSPDM_STATUS_SUCCESS
LIBSPDM_STATUS_INVALID_PARAMETER
SPDM_ERROR_CODE_VERSION_MISMATCH
SPDM_ERROR_CODE_INVALID_REQUEST
SPDM_ERROR_CODE_UNEXPECTED_REQUEST

Should I add these exact ones instead, or delete all retval fields instead?
Thank you.

Reply:
The `RETURN_*` text needs to be deleted throughout libspdm. It's just been put off in favor of more pressing issues.